presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -365,6 +366,8 @@ public void testDecimalExceedingPrecisionMaxConvertedToVarchar()
                 "'12345678901234567890123456789012345678901234567890'");
     }
 
+    // "Unsupported type handling is set to CONVERT_TO_VARCHAR, but toPrestoType() returned empty"
+    @Flaky(issue = "https://github.com/prestosql/presto/issues/4885")
     @Test
     public void testDecimalExceedingPrecisionMaxWithExceedingIntegerValues()
```

Comment:
Did anyway actually tried to investigate this? It seems similar to PSQL issue. I suspect it could be faulty test infra (query runner) code somewhere and it could strike back in different test.

Reply:
i know two similarily named test methods are affected for sure.